### ISSUE TITLE
Exit option in system tray does not work in NormCap-0.4.3-x86_64.AppImage

### ISSUE THREAD

**Issue by SnowballV** [dynobo/normcap#458]
### What happened?

Using NormCap-0.4.3-x86_64.AppImage on Kubuntu 22.04.2, the NormCap Exit option in Kubuntu system tray does not exit NormCap, but instead it opens the capture window.
Only when I disable "Keep in system tray", I can close and exit NormCap-0.4.3-x86_64.AppImage.
This behavior is new in NormCap-0.4.3-x86_64.AppImage, it was not in previous versions.

### How did you install NormCap?

AppImage (Linux)

### Operating System + Version?

Kubuntu 22.04.2

### [Linux only] Display Server (DS) + Desktop environment (DE)?

DS: Xorg, DE: KDE Plasma

### Debug log output?

```shell
23:34:36 - INFO    - normcap:30 - Start NormCap v0.4.3
23:34:36 - DEBUG   - normcap.gui.tray:60 - System info:
{'cli_args': '/tmp/.mount_NormCaVvla5G/usr/app/normcap/__main__.py -v debug', 'is_briefcase_package': True, 'is_flatpak_package': False, 'platform': 'linux', 'pyside6_version': '6.5.1', 'qt_version': '6.5.1', 'qt_library_path': '/tmp/.mount_NormCaVvla5G/usr/app_packages/PySide6/Qt/plugins, /tmp/.mount_NormCaVvla5G/usr/python/bin', 'config_directory': PosixPath('/home/xxxxx/.config/normcap'), 'normcap_version': '0.4.3', 'ressources_path': PosixPath('/tmp/.mount_NormCaVvla5G/usr/app/normcap/resources'), 'tesseract_path': PosixPath('/tmp/.mount_NormCaVvla5G/usr/bin/tesseract'), 'tessdata_path': PosixPath('/home/xxxxx/.config/normcap/tessdata'), 'envs': {'TESSDATA_PREFIX': None, 'LD_LIBRARY_PATH': None}, 'desktop_environment': <DesktopEnvironment.KDE: 2>, 'display_manager_is_wayland': False, 'screens': [Screen(is_primary=True, device_pixel_ratio=1.0, rect=Rect(left=0, top=0, right=1920, bottom=1080), index=0, screenshot=None)]}
23:34:36 - DEBUG   - normcap.gui.tray:342 - Listen on local socket v0.4.3-normcap.
23:34:36 - DEBUG   - normcap.gui.settings:128 - Skip update of non existing setting (cli_mode: False)
23:34:36 - DEBUG   - normcap.gui.settings:128 - Skip update of non existing setting (background_mode: False)
23:34:36 - DEBUG   - normcap.screengrab:32 - Select capture method QT
23:34:36 - DEBUG   - normcap.gui.utils:22 - Save debug image as /tmp/normcap/1685828076.9509819_raw_screen0.png
23:34:37 - DEBUG   - normcap.gui.window:131 - Create window for screen 0
23:34:37 - DEBUG   - normcap.gui.window:193 - Set window of screen 0 to fullscreen
23:34:37 - DEBUG   - normcap:216 - [QT] qtwarningmsg - qsystemtrayicon::setvisible: no icon set
23:34:37 - DEBUG   - normcap.ocr.tesseract:23 - Tesseract command output:
List of available languages in "/home/xxxxx/.config/normcap/tessdata/" (1):
eng
23:34:48 - DEBUG   - normcap.gui.tray:294 - Hide 1 window
23:35:07 - DEBUG   - normcap.screengrab:32 - Select capture method QT
23:35:07 - DEBUG   - normcap.gui.utils:22 - Save debug image as /tmp/normcap/1685828107.2169085_raw_screen0.png
23:35:07 - DEBUG   - normcap.gui.window:131 - Create window for screen 0
23:35:07 - DEBUG   - normcap.gui.window:193 - Set window of screen 0 to fullscreen
23:35:33 - DEBUG   - normcap.gui.tray:294 - Hide 1 window
23:36:13 - DEBUG   - normcap.screengrab:32 - Select capture method QT
23:36:13 - DEBUG   - normcap.gui.utils:22 - Save debug image as /tmp/normcap/1685828173.2543898_raw_screen0.png
23:36:13 - DEBUG   - normcap.gui.window:131 - Create window for screen 0
23:36:13 - DEBUG   - normcap.gui.window:193 - Set window of screen 0 to fullscreen
23:36:42 - INFO    - normcap.gui.tray:503 - Exit normcap (clicked close in menu)
23:36:42 - DEBUG   - normcap.gui.tray:504 - Debug images saved in /tmp/normcap

I had to disable "Keep in system tray" to be able to close and exit NormCap-0.4.3-x86_64.AppImage.
```

**Comment by dynobo** [comment]
Thanks for reporting, @SnowballV! This is clearly a regression, I'm looking into it!

**Comment by dynobo** [comment]
Should be fixed with [v0.4.4](https://github.com/dynobo/normcap/releases). If not, please reopen.
Thanks again for reporting! :slightly_smiling_face:

**Comment by SnowballV** [comment]
The reported issue is fixed in NormCap-0.4.4-x86_64.AppImage.
Thank you very much!
Also thank you for the application, I love it.
